index.js

```diff
@@ -21,25 +21,25 @@ const port = process.env.PORT || process.env.LOCAL_PORT;
 // Create global app object
 const app = express();
 
-app.use(session);
 app.use(cors());
 
 // Normal express config defaults
 app.use(require('morgan')('dev'));
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
+sessionManagement(app);
 
 app.use(express.static(`${__dirname}/public`));
 app.use(auth.initialize());
-app.use('/api/user', userRoute);
+app.use('/api/users', userRoute);
 
 app.get('/', (req, res) => res.status(200).send({
   status: 'connection successful',
   message: 'Welcome to Author Haven!',
 }));
 
-app.listen(port, function serverListner() {
+app.listen(5000, function serverListner() {
```

Comment:
@un-tethered Please revert back to `port` in the `app.listen` section of the code.